sfaira/unit_tests/tests_by_submodule/ui/test_userinterface.py

```diff
@@ -17,30 +21,62 @@ class HelperUi:
     basic_estimator_test(). See _test_call() for an example.
     """
 
-    def simulate(self):
-        """
-        Simulate basic data example used for unit test.
+    def __init__(self):
+        self.temp_fn = os.path.join(DIR_TEMP, "test_data")
+
+    def prepare_local_tempfiles(self):
+        # create temp directory
+        if not os.path.exists(self.temp_fn):
+            os.makedirs(self.temp_fn)
+        # download an example weight from sfaira repo
+        lookuptable = pd.read_csv(os.path.join(SFAIRA_REPO_URL, 'model_lookuptable.csv'), header=0, index_col=0)
+        url = lookuptable.loc[0, "model_file_path"]
+        if os.path.basename(url) not in os.listdir(self.temp_fn):
+            urllib.request.urlretrieve(url, os.path.join(self.temp_fn, os.path.basename(url)))
 
-        Sets attribute .data with simulated data.
+    def _get_adata(self):
+        """
+        Create an adata object for use in unit tests
 
         :return:
         """
-        pass
+        dsg = prepare_dsg(rewrite=True, load=False)
+        dsg.subset(key="id", values=["human_lung_2021_None_mock4_001_no_doi_mock4"])
+        dsg.load()
+        return dsg.adata
```

Comment:
You might want to (variably) take out some annotation here so that you can see further down works on minimally annotated data (the usual input). In addition, it would probably make sense to also check that the unit tests on DatasetInteractive cover failure modes that you do not necessarily need to check in a unit test here, that loads and executes model ie which are likely relatively long unit tests?

Reply:
I'm already running this with kind of minimal annotation, not sure if it makes sense to take out any more than that:
```
AnnData object with n_obs × n_vars = 20 × 60
    obs: 'dataset'
```